erpnext/maintenance/doctype/maintenance_visit/maintenance_visit.py

```diff
@@ -16,8 +17,34 @@ def validate_serial_no(self):
 			if d.serial_no and not frappe.db.exists("Serial No", d.serial_no):
 				frappe.throw(_("Serial No {0} does not exist").format(d.serial_no))
 
+	def validate_mntc_date(self):
```

Comment:
Don't write function names based on field names. Noone will know off the bat what `mntc` is. 
```suggestion
	def validate_maintenance_date(self):
```